Sources/HaCWebsiteLib/routes.swift

```diff
@@ -13,23 +13,16 @@ func getWebsiteRouter() -> Router {
 
   router.get("/", handler: LandingPageController.handler)
 
-  router.get("/workshops") { request, response, next in
-    defer {
-      next()
-    }
-    do {
-      try response.send(
-        UI.Pages.workshops(workshops: WorkshopManager.workshops).render()
-      )
-      .end()
-    } catch {
-      Log.error("Failed to render template \(error)")
-    }
+  router.get("/workshops") { _, response, next in
+    try response.send(
+      UI.Pages.workshops(workshops: WorkshopManager.workshops).render()
+    ).end()
+    next()
   }
 
   /// Intended for use by GitHub webhooks
-  router.post("/api/refresh_workshops") { request, response, _ in
-    try! WorkshopManager.update()
```

Comment:
Willing to discuss removing the force try here, in which case we can leave it out of the PR and come back to the idea later.

Reply:
So am I right in thinking that with the force try, it would crash if `WorkshopManager` crashes, and without it, it would use a potentially out of date set of workshops?

Reply:
Correct. We can add logging here and/or tests on the workshop manager repo to catch this stuff. I'm all for failing fast, but in this case an update to a GitHub repo can crash our app automatically. That's some spooky action at a distance right there.

Reply:
👍